### ISSUE TITLE
[BUG] pivpn.sh was named incorrectly

### ISSUE THREAD

**Issue by p1r473** [pivpn/pivpn#1302]
## In raising this issue, I confirm the following: 

- [X] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting *is* directly related to the pivpn installer script.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

**Describe the bug**
After installing, I could not use `pivpn`
![image](https://user-images.githubusercontent.com/9235633/117327623-ce875080-ae60-11eb-8d37-f0dfa27f278c.png)
I chased the symlink in `/usr/local/bin`
![image](https://user-images.githubusercontent.com/9235633/117327688-e19a2080-ae60-11eb-82b3-ffbb49f908cc.png)
I realized the symlink was invalid and was pointing to /usr/local/src/pivpn/scripts/pivpn.sh
However, the `pivpn.sh` script was named `pivpn`
![image](https://user-images.githubusercontent.com/9235633/117327838-08585700-ae61-11eb-87ee-c69daa71341a.png)
So, I renamed `pivpn` to `pivpn.sh`
![image](https://user-images.githubusercontent.com/9235633/117328322-8583cc00-ae61-11eb-92d9-2fbf138a6e52.png)
-Symlinked is now fixed
![image](https://user-images.githubusercontent.com/9235633/117327936-21f99e80-ae61-11eb-9428-eb01046a0e3f.png)
-can now use `pivpn`
![image](https://user-images.githubusercontent.com/9235633/117327983-2d4cca00-ae61-11eb-857d-83301744f088.png)

TLDR: 
Installer script had an issue: `/usr/local/src/pivpn/scripts/pivpn.sh` was named `pivpn` and not `pivpn.sh`
and this caused the pivpn symlink to not work

**To Reproduce**
Installed with `curl -L https://install.pivpn.io | bash`

**Expected behavior**
pivpn should work
`/usr/local/src/pivpn/scripts/pivpn.sh` should be named `pivpn.sh` and not `pivpn`

**Comment by p1r473** [comment]
Proposed a fix in https://github.com/pivpn/pivpn/pull/1303

**Comment by p1r473** [comment]
Noticing the same issue for the `bash-completion` files
![image](https://user-images.githubusercontent.com/9235633/117328750-ed3a1700-ae61-11eb-98dc-f900bbcf67b0.png)

No extension:
![image](https://user-images.githubusercontent.com/9235633/117328681-dabfdd80-ae61-11eb-883f-f5c1821a0378.png)

**Comment by 4s3ti** [comment]
I don't think there is a bug here,
Those scripts are named like that intentionally and have a purpose. 

Looking at the code and how the scripts are installed: https://github.com/pivpn/pivpn/blob/master/auto_install/install.sh#L2259-L2292 
Everything seems to be as expected and i don't see anywhere in the code the behavior you describe. 

On top of that, we have CI scripts testing the installation of PiVPN: https://github.com/pivpn/pivpn/blob/master/.travis.yml and those would have signaled it long time ago. 

Regardless of that and just to be really sure 

I used your steps "how to replicate", and based on your screenshots guessed that you are using OpenVPN as VPN protocol.

And everything seems to be working as expected. 

```
root@testserver-0:~# pivpn 
::: Control all PiVPN specific functions!
:::
::: Usage: pivpn <command> [option]
:::
::: Commands:
:::  -a, add [nopass]     Create a client ovpn profile, optional nopass
:::  -c, clients          List any connected clients to the server
:::  -d, debug            Start a debugging session if having trouble
:::  -l, list             List all valid and revoked certificates
:::  -r, revoke           Revoke a client ovpn profile
:::  -h, help             Show this help dialog
:::  -u, uninstall        Uninstall PiVPN from your system!
:::  -up, update          Updates PiVPN Scripts
:::  -bk, backup          Backup Openvpn and ovpns dir
```


All this being said  What i see here is rather a system you have been tinkering with .. please be more specific with the steps on how to replicate this or retry on a clean system please. 

If this would be a bug we would have had more complains and issues long time ago.

**Comment by 4s3ti** [comment]
Might also be worth trying test branch as suggested here: 

https://github.com/pivpn/pivpn/issues/1264#issuecomment-791925393
